controller/controller.go

```diff
@@ -471,9 +472,14 @@ func updateOtherConfigMaps(c *client.Client, oc *oclient.Client, svc *api.Servic
 // findOAuthAuthorizeURL uses this endpoint: https://github.com/openshift/origin/pull/10845
 func findOAuthAuthorizeURL() string {
 	url := "https://openshift.default.svc/.well-known/oauth-authorization-server"
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
```

Comment:
should we make this configurable? when running on a prod system I expect we'll want this to be false?

Reply:
Yeah good idea, I didn't think it was something we need since it's only for internal cluster communication but thinking about it more maybe there is some case with multi tenancy we may want to make it extra sure on production system